### PR TITLE
New homepage adjustments

### PR DESCRIPTION
**Pull request by ghost**
Make some changes and adjustments suggested by the team to the new homepage.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -36,27 +36,18 @@ These topics are the most visited ones, we thought you’d like to have a glance
     <a class="tp-content-card" href="manuals/reference">
       <img class="tp-content-card-icon icon-light" src="images/icons/menu_book-w.svg">
       <img class="tp-content-card-icon icon-dark" src="images/icons/menu_book.svg">
-      <h3>Reference Manual</h3>
+      <h3>User Manual</h3>
```
Comment (jrobinAV) [inline docs/index.md:39]: Either the title or the description is incorrect. 
The documentation for all APIs exposed in Taipy is the Reference manual.
The user man is more a functional documentation with standard examples.

File: docs/index.md [hunk 2]
```diff
@@ -36,27 +36,18 @@ These topics are the most visited ones, we thought you’d like to have a glance
...
-      <p>Controls are UI objects for data vizualisation and interaction.</p>
-      <span class="tp-content-card-readmore">Read more</span>
-    </a>
-  </div>
   <div class="tp-col-12 tp-col-md-6 tp-col-lg-4 d-flex">
     <a class="tp-content-card" href="manuals/about/">
       <img class="tp-content-card-icon icon-light" src="images/icons/dashboard-w.svg">
       <img class="tp-content-card-icon icon-dark" src="images/icons/dashboard.svg">
-      <h3>Taipy’s user interface</h3>
+      <h3>Taipy User Interface</h3>
```
Comment (FabienLelaquais) [inline docs/index.md]: I suspect you don't want "Taipy".
Or, you want it everywhere, which I doubt.

**Review by FabienLelaquais — APPROVED** [review]
With a minor yet relevant change...

File: docs/index.md (hunk 1 above)
Comment (ghost) [reply docs/index.md:39]: It has been asked from me just to replace the title... I wasn't sure it had been discussed with you guys from R&D beforehand, but I was told it did.
I'll check if I misunderstood the demand or else...

File: docs/index.md (hunk 2 above)
Comment (ghost) [reply docs/index.md]: Yup, we can get rid of that, I guess.

File: docs/index.md (hunk 1 above)
Comment (jrobinAV) [reply docs/index.md:39]: ok, thx.

**Review by jrobinAV — APPROVED** [review]

**Pull request closed without merging**